### ISSUE TITLE
[ui/ux][icon] Unify the icon color settings for Menu and Toolbar

### ISSUE THREAD

**Issue by BroKun** [eclipse-theia/theia#5659]
### Description
Use the color property on Menu via .p-Menu-item .p-mode-disabled
Use the opacity property on TabBar-Toolbar via .p-TabBar-toolbar .item.enabled

I think the opacity setting is more friendly for color icons, I don't know if there are any other considerations.

**Comment by akosyakov** [comment]
Could you add screenshots to show how it is different now? PRs are welcomed as usual. I've nominated you as a Theia committer, as soon as it is done we can add you add as a collaborator that you can triage issues, be assigned to reviews and so on.

**Comment by BroKun** [comment]
> Could you add screenshots to show how it is different now? PRs are welcomed as usual. I've nominated you as a Theia committer, as soon as it is done we can add you as a collaborator that you can triage issues, be assigned to reviews and so on.

Thank you so much, it's my pleasure.

**Comment by BroKun** [comment]
For colored icons, the use of color attribute settings on menu is invalid, but elsewhere we use opacity settings, so it works.
![20190708200121](https://user-images.githubusercontent.com/11978904/60816003-5354a500-a1cb-11e9-9990-2073ecd88076.jpg)
![20190708201305](https://user-images.githubusercontent.com/11978904/60816007-564f9580-a1cb-11e9-867d-cedd27b9f8f8.jpg)
![20190708213914](https://user-images.githubusercontent.com/11978904/60816016-58b1ef80-a1cb-11e9-9e15-ae340b3299af.jpg)

**Comment by BroKun** [comment]
![opacity-search](https://user-images.githubusercontent.com/11978904/65013638-77143400-d94e-11e9-939c-ebd0e735da21.jpg)

When I looked at this problem, I found some strange places. For example, in this area on the map, we set a lot of opacity values, which is very confusing.

We added 'theia-mod-disabled' style very early, the value used here is 0.4, maybe we can set this value to a css variable, I will unify these values ​​first.
